Sprint 9

ulia-lukash opened the pull request.

ulia-lukash committed:
Sprint 9

So1ovyov commented on ImageFeed/ImageList/ImagesListViewController.swift:
```diff
@@ -8,25 +8,41 @@
 
 final class ImagesListViewController: UIViewController {
+    
+    private let ShowSingleImageSegueIdentifier = "ShowSingleImage"
```
Наименование констант и переменных должно быть в стиле CamelCase и начинаться с нижнего регистра.

So1ovyov commented on ImageFeed/ImageList/ImagesListViewController.swift:
```diff
@@ -8,25 +8,41 @@
+            let indexPath = sender as! IndexPath
+            let imageName = photosName[indexPath.row]
+            let image = UIImage(named: imageName)
+//            _ = viewController.view
```
Закомментированный код, можно удалить.

So1ovyov commented on ImageFeed/ProfileViewController.swift:
```diff
@@ -0,0 +1,129 @@
+import Foundation
+import UIKit
+
+
```
Лишняя пустая строка, можно удалить.

So1ovyov commented on ImageFeed/ProfileViewController.swift:
```diff
@@ -0,0 +1,129 @@
+        private func logOut() {
+            
+            }
+    
```
Лишняя пустая строка, можно удалить.

So1ovyov commented on ImageFeed/ProfileViewController.swift:
```diff
@@ -0,0 +1,129 @@
+    }
+    
+    @objc
+        private func logOut() {
```
Отступ сместился

So1ovyov commented on ImageFeed/SingleImageView/SingleImageViewController.swift:
```diff
@@ -0,0 +1,70 @@
+import Foundation
+import UIKit
+
+
```
Лишняя пустая строка, можно удалить.

So1ovyov commented on ImageFeed/SingleImageView/SingleImageViewController.swift:
```diff
@@ -0,0 +1,70 @@
+extension SingleImageViewController: UIScrollViewDelegate {
+    func viewForZooming(in scrollView: UIScrollView) -> UIView? {
+            imageView
+        }
```
Отступ сместился

ulia-lukash committed:
Sprint 10

ulia-lukash committed:
sprint 11

ulia-lukash committed:
Не получается получать URL аватарки на SplashScreen'е - вынесла в ProfileView.
Почему-то при создании TabBarItem'ов в коде те же самые картинки располагаются выше, чем, если бы было сделано через Storyboard.

ulia-lukash committed:
feat: ImageListService

ulia-lukash committed:
fix: nil token at photos fetch before login in

ulia-lukash committed:
fix: SingleImageView segue preparation

ulia-lukash committed:
Merge pull request #7 from ulia-lukash/sprint_12

Sprint 12

ulia-lukash committed:
Merge pull request #6 from ulia-lukash/sprint_11

Sprint 11

ulia-lukash committed:
Merge pull request #5 from ulia-lukash/sprint_10

Sprint 10
